x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/schema/schema.tsx

```diff
@@ -77,7 +76,11 @@ export const Schema: React.FC = () => {
   );
 
   return (
-    <>
+    <SourceLayout
+      pageChrome={[NAV.SCHEMA]}
+      pageViewTelemetry="source_schema"
+      isLoading={dataLoading}
+    >
```

Comment:
NOTE: The Schema page could *possibly* use the new `isEmptyState`/`emptyState` page template props, but would need some layout reshuffling.

TBH the Source pages might need to be rethought a bit generally, their custom source header made it so I couldn't make use of EuiPageTemplate's `pageHeader` prop generally at all in this PR